physutils/io.py

```diff
@@ -8,13 +8,124 @@
 import os.path as op
 
 import numpy as np
+from bids import BIDSLayout
 from loguru import logger
 
 from physutils import physio
 
 EXPECTED = ["data", "fs", "history", "metadata"]
 
 
+def load_from_bids(
+    bids_path,
+    subject,
+    session=None,
+    task=None,
+    run=None,
+    extension="tsv.gz",
+    suffix="physio",
+):
+    """
+    Load physiological data from BIDS-formatted directory
+
+    Parameters
+    ----------
+    bids_path : str
+        Path to BIDS-formatted directory
+    subject : str
+        Subject identifier
+    session : str
+        Session identifier
+    task : str
+        Task identifier
+    run : str
+        Run identifier
+    suffix : str
+        Suffix of file to load
+
+    Returns
+    -------
+    data : :class:`physutils.Physio`
+        Loaded physiological data
+    """
+
+    # check if file exists and is in BIDS format
+    if not op.exists(bids_path):
+        raise FileNotFoundError(f"Provided path {bids_path} does not exist")
+
+    layout = BIDSLayout(bids_path)
```

Comment:
Not sure if there is a reason not to do that but I think we should specify `layout = BIDSLayout(bids_path, validate=False)`. I tested the current function with a BIDS dataset containing multiple physio files saved under names using different label values with the `recording` entity (ex. <match>_recording-pulse_<match> and <match>_recording-breathing_<match> ), and if `validate=False` is not specified, it just gives me an empty list after `layout.get()`

Reply:
This change + the addition of the recording parameter would allow people with BIDS dataset containing physio data with different sampling rate to still be able to use those functionalities